dan_lombardy/index.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+var http = require('http');
+var app = require('express')();
+var fs= require('fs');
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+app.post("/unicorns", function(req, res, next){
+  var data = JSON.stringify(req.body);
+  fs.writeFile(__dirname + "/data/unicorns_one.json", data, function(err){
```

Comment:
I think an important part of the assignment was to make it so POST and GET requests could be made to any route specified by the user and save or access a file with that route's name (e.g., POST to /x makes x.json and GET to /x reads x.json).